extensions/interactions/MathExpressionInput/directives/oppia-interactive-math-expression-input.directive.ts

```diff
@@ -148,8 +148,18 @@ angular.module('oppia').directive('oppiaInteractiveMathExpressionInput', [
             });
           };
 
+          ctrl.getLatexErrorText = function() {
+            if (guppyInstance.latex().includes('=')) {
+              return $translate.instant('I18N_MATH_EXPRESSION_ERROR');
+            }
+            return '';
+          };
+
           ctrl.isCurrentAnswerValid = function() {
             var latexAnswer = guppyInstance.latex();
+            if (latexAnswer.includes('=')) {
```

Comment:
Hmmmm, I doubt this will just work. Think of the following:
- What do you do with explorations which already have an expression where they have set an answer group with = in it.
- How do you handle such explorations? What should the policy be?
- AFAIK, one of the demo explorations (all_interactions) uses the math expressions interaction with the desired output as y=mx+c. That will become unplayable.

Reply:
Some points about the first bullet... You can first run an audit job to check how many such interactions exist, then decide the further course of action. This change should ideally be the last step of the fix.

Reply:
@nithusha21 This fix was decided so that expressions are used instead of equations. You're right about the "y=mx+c" part, I've addressed it [here](https://github.com/oppia/oppia/issues/5296#issuecomment-583705252) as editing that interaction was a part of fixing this issue. PTAL.

Reply:
What about other existing explorations (if any) with the math expressions input with an = sign in it? They wouldn't work anymore right?

Reply:
@nithusha21 I have created a [PR](https://github.com/oppia/oppia/pull/8669) with an audit job that would surface all such explorations.